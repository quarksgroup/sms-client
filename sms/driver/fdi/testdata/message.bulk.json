{
  "success": true,
  "message": "Message successfully queued for 2 out of 3 numbers",
  "data": {
    "valid": 20,
    "invalid": 5
  },
  "cost": 2,
  "msgRef": "30bb083a-ae95-43b9-8ed5-051693d018af",
  "gatewayRef": "3e381e5d-3dfe-4b27-99d9-6e6f8bcdce30"
}