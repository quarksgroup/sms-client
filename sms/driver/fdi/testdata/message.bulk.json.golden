{
  "ID": "30bb083a-ae95-43b9-8ed5-051693d018af",
  "Cost": 2
}